doc/dev/cephfs-snapshots.rst

```diff
@@ -53,6 +53,17 @@ To create a CephFS snapshot, create a subdirectory under
 ``.snap`` with a name of your choice. For example, to create a snapshot on
 directory "/1/2/3/", invoke ``mkdir /1/2/3/.snap/my-snapshot-name`` .
 
+.. note::
+   Snapshot names can not start with an underscore ('_'), as these names are
+   reserved for internal usage.
```

Comment:
Maybe we should mention that when creating a snapshot, the snapshot name cannot start with an underscore ('_')... ? If in one `.snap/` when you see the snapshot names starting with '_', they must be inherit from the parent snap realms.

Reply:
Ok, I'll rephrase to make that clear and to mention the parent snap realms names.  Thanks!